lib/ruby_vm/mjit/c_pointer.rb

```diff
@@ -219,7 +219,7 @@ def self.[]=(addr, value)
     end
 
     class Pointer
-      attr_reader :type
+      attr_reader :type, :addr
```

Comment:
How about calling it `to_i`? I feel it'd be less likely to conflict with existing names used in C, and that's why I already use the name for the same functionality in `class Immediate` (which is a pointer of an immediate and thus has `@addr`).

Reply:
It's also _kind of_ useful that `nil.to_i` becomes 0.

Reply:
ah yes, I totally agree.  It makes sense.  I've updated the patch to use `to_i`.